website/docs/components/table/partials/code/component-api.md

```diff
@@ -41,9 +53,85 @@ The Table component itself is where most of the options will be applied. However
     If set, determines the vertical alignment for all cell (`td`) content in a table. Does not apply to table headers (`th`).
   </C.Property>
   <C.Property @name="caption" @type="string">
-    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text.
+    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text. Note: it is best practice to provide a table caption for users with assistive technology.
   </C.Property>
   <C.Property @name="...attributes">
     Supported for the `Hds::Table` component.
   </C.Property>
+  <C.Property @name="onSort" @type="function">
+    Callback function that is invoked when one of the sortable table headers is clicked (or has a keyboard interaction performed). The function receives the values of `sortBy` and `sortOrder` as arguments.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::Tr
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of the `Th`, `ThSort`, or `Td` elements.
```

Comment:
I am still not convinced of these phrase here (and also in the other sub-components below).
This content is the "Component API" while what we're describing here is "how to use" it, which is different. 
Maybe should go under the "how to use" block, or in the accessibility section?

Reply:
I'll double check this.